css/toolbar.css

```diff
@@ -0,0 +1,22 @@
PLL01P78fMr8vP78dO78bOb8TM78jPr8LKr0bMb8XN783PL8RMr8rP789P78VNb8PML8vOb8rN785P78////AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB/6AS4KDhIWGh4iJiouMjY6PkJGShEVKk5eYmZqSGhk7BTWboqOknCVGBSSpJaWtrq8qK6mzBTcZr7i5k0UZPQW0N78FDbrFxohFREOzqsHMOsfRxhTKv83C1wUq0tyvOr6qJM7ZqTfb3eibGp/hzuXWtb846fSXRBiz7qrY4wUw9QAdKXmwT188ccJoWQrIEJEKfLTaJWR2cGHDi0solJCYj987fUUwXizCziO2k9geNAgpkqESEe8QwkMokwQGHxZbBoTha6K1YO6sYchgQudFGEMkykx4bShLowxxQNxHNRhHHxqgXsRxpGM8kwUe/NPaECmtiFaxlaBAtmERVP40rQYNJmJs24AUHpz1KXPF07v1KOwoGDPtKsANS8BjKswqMcQBFXSMS+uIAqMDgBwQqQHcxH3xjswzigLADxlsGw4+KBeeZa0dAACI0UEIQx2Mz1o9Z9TEbNkAJCQBqCHpV6BnH0NNAjwG8Bap0UWYOdGqj7YTnDeXDaFoNxzvDob75SC6UQoSfmv/fWIDt7woZQbrYaPtDODbZccIsTlaA9af/cKbVgE4Z2B+ANzQXzEUGAFaYalcdxcE+Om3XQwneKcLbjH5JMJfWs2QwIEW4hcDBOa5QoEFunV4GWImjFCies2hsGGHTN2wA2SDVKDder8Bh0QuPCTUWjg5Qf42ww8W/vjbDy+8AkN4LUrI4yAXsGDilgS8YsBJ/aSS1ZWECKHlekAC4EIrReDTD00FWEnmIB842aRp9ZGCG0eNFZDknEskgWaFMdhICgi/6JOWA4AekkOQNAa3oCYUHEHVV9YMSMikiBVxwoz4GbqJCokuhlAPKQpywA/DXTnAlgdKwOklEYznzDhyEhKAbDmkih2kFQYgCkxfXVrAaIVQwIJ2ELgH2QEHRgtACL5CosR4k92A6iEfVMjCADyWVmFzKWjCYWu05DrIBJHGIANkL0i73gSa+LBXRNAYYgKT4wIwgYZttTCugTFU64heR946ZiFIyAvcDIjVGSlw4P5G0klGXd2rihGIiEsoAAnweMPHznkASRE1HHEEBdeG1xoHiHwqrWwI8OjBxLJ14EgRJUwFA6mgYVsAEYf4luZvECtJsmwGC8LLMrSokAGAxb5YyKt3yvZD0zpR8AOJzVV8CDWeIUeEDw9WVcDChMgArIEjkEkArLJVMHYDcBX7iwHsoCuMr4+CCoCwVyIgOAA5jM3RpQ4U+ZlVPSAyt5PaMUAmA0FGG/chPcGDHA8w+S1OEDH361zSPN7XLwA6H4JwePuI0AO22TB6SAg0rgcwYgfk/iQiDgg9HgZQ6/3LCoiYzvScR88WAyKflIpQMBiQUNNBBcB8iO/OA5oezv6IcHA9Sp6fpH0hvn1sGqBa0r27IBwsZpj1sB90PiEU4ByDBIDGADZwvppO/ZyRMeocDxGD+hGglva8Q8iiH/2oHrqssqNDpCdNzuFab3AWHOgZKx8Y6EnQZsEDRCTgQsCJ0pVesLQbIKJI11hKDyxAGPmQ7hAUwiAAPkAmiR0NAohARVDgYQEYLmYWGEAEECgHHLtdqQIodA4QEGGpbJGnVsd5hxF8dbOBAaAFZGpB82JgMkMUYTL8AIFiKPKLHjSgWpibGQBOQKZPcc9yhpiSQdxRggZkwyZEAFEhNqA8ALyPLL0DVgoP8R9mvKkBCqCKUxpxJkg5p1WIwVwUf/6AiBoQpnwKOOMqBJkIGmxJP5tDjIyal8pCOEBvNclKDwzAtkZAcVD6OWRvmGggJyYrY59MxRCcNglCWvI3vrRP82TjrEKAh0/NqCAmYqPIOSppRLmj4yEy8MfwsCITbruTdsqFGBvMLVgvRIk+rHYJaB3NOa1DDAUMd6BZLaEISqEOKSPBLg7iETIp0E4rCUGEFsmkhJtAwtKmpUutvOBTPDxE9AzTmFuIogP/M5CoIGOCjRJCA0JD158ukYJTNieijSKEYuAUNI6RIjZMlA0L8pTSjHQui6r45iga5kXnEGCfPNoTLONhl1Gw63/AmWJKVwRLdzzAFauK4m8S1/4oDjmSH8opxS1XJ5sFAGpFtvLIEICqCRn5bl5khcp/lrIXA+TCBCNTHjztqZUiFO+qSNLFDPzHQZn+8y5fusZcCoC8YjCHgcChalskM76aFDUXPqqmeujVliIIYIRLIQEIpLEr7gHnr1ABwScNQoKR5sJwDKQpVHTQzavqVBqCqiYQyQKDAnLEGT1IKy4C6ruaaUUDAoBQRAqQr3RcIK4n1Qp8PlgqwgbEBEt8kgbpIZhs3Gp4uj0GA9qn1K7Jgo1BCwY7n4uCGGBSJNV1mfR+oS6GXKCZGKHAdzHbEQtMt6ai0EAQ1JvFAhzhsfjFBU/OYiyqaCrAr2hAAScYHuOLIhgXJPnJEdPW3gePQgWekR+EymNhVygBURBibgE43GFS8KyKD5LerUZ83xIzIhmza25aUqyKFbTYxYng2TJoJ2HsBcMHN8axIRQAgq7UkKJ4JZqQJ0GBGniGHH0aXw/Gu+RHaAAt5SuwMx5Qyyo/omVDDCtbSQBkL18CBpey7hHnc2AzP+KZ8QhTDQtggOy6GRGRDNCRfmEBZN15EqSyCpSFgQEiBPnPhWAtlSRSaDsjOhFrBcskH62Jgi7aAiuhtCimJkkf+FnTmlDMDUiwAx00FNSTyIABVOBoVLv61UsIBAA7') no-repeat center;
```

Comment:
:) 
It is a loading anigif shown in the center of the page when clearing the cache runs from the admin bar and disappears when the process is complete.
But I'll make is loaded externally from a .gif file to modify it easily until you set which is the most appropriate.